lib/rubocop/directive_comment.rb

```diff
@@ -88,10 +88,15 @@ def cop_names
       @cop_names ||= all_cops? ? all_cop_names : parsed_cop_names
     end
 
+    # Returns an array of cops for this directive comment, without resolving departments
+    def raw_cop_names
```

Comment:
I guessing this should stay private.

Reply:
No, it needs to be accessed by the comment config. I've added a test, but it would also be fine to mark it `@api private` if you prefer.

Reply:
Ah, yeah - just notice that usage in the file above. I'm not sure about the name, but it's not a big deal. We can consider it public API.